source/about.html.erb

```diff
@@ -32,16 +32,14 @@ title: "A framework for creating ambitious web applications."
 
 <div id="download">
   <div id="download-ember">
-    <a class="orange button" href="https://github.com/emberjs/starter-kit/archive/v1.9.1.zip" download>
-      Download the Starter Kit
+    <a class="orange button" href="/guides/ember-cli/installing">
+      Install Ember.js
     </a>
     <div class="info">
-      1.9.1:
-      <a class="debug" href="http://builds.emberjs.com/tags/v1.9.1/ember.prod.js" download>production</a>
-      <a class="debug" href="http://builds.emberjs.com/tags/v1.9.1/ember.min.js" download>(min + gzip  97kb)</a> |
-      <a class="debug" href="http://builds.emberjs.com/tags/v1.9.1/ember.js" download>debug</a> |
-      <a href="http://builds.handlebarsjs.com.s3.amazonaws.com/handlebars-v2.0.0.js" download>Handlebars</a> |
-      <a class="debug" href="/guides/getting-ember">packages</a>
+      With <a href="http://nodejs.org/download/" target="_blank">node.js and npm</a> installed, you can get started by simply running:
```

Comment:
Do we need to say "With node and npm" installed? I think the vast majority of our audience will assume this. I think I'd rather have a treatment à la ember-cli.com with just the `npm install -g ember -cli` command, with perhaps a small link underneath that says "installation guide" that goes through the dependencies required in depth, as well as common troubleshooting steps.